src/engraving/dom/score.h

```diff
@@ -877,7 +877,8 @@ class Score : public EngravingObject, public muse::Injectable
     void transposeDiatonicAlterations(TransposeDirection direction);
 
     struct InsertMeasureOptions {
-        InsertMeasureOptions() {}
+        InsertMeasureOptions() {
+        }
```

Comment:
Another unrelated change (probably caused by uncrustify I guess; it might depend on which version you use, we use version 0.73).